backend/src/api/SiteController.ts

```diff
@@ -107,6 +107,7 @@ export default class SiteController {
         }
         catch (error) {
             this.logger.error('Could not change subscription', { site, main, bookmarks, error });
+            this.logger.error(error);
```

Comment:
Forgot to remove debug? Double logging `error`

Reply:
Nope, this is intentional. The first line doesn't print the stacktrace, the second does.